Horizon/Horizon/Sources/Features/Notebook/Common/View/NoteBookLabelIcon.swift

```diff
@@ -21,25 +21,20 @@ import SwiftUI
 struct NotebookLabelIcon: View {
     // MARK: - Dependencies
 
-    let type: NotebookNoteLabel
-    let enabled: Bool
+    let type: CourseNoteLabel
 
     var body: some View {
         let image = type == .confusing ?
             Image(systemName: "questionmark.circle") :
             Image(.flag)
-        return image.foregroundStyle(enabled ? colorFromType(type) : Color.disabledGray)
+        return image.foregroundStyle(colorFromType(type))
     }
 }
 
 // MARK: - Helpers
 
-@inline(__always) func colorFromType(_ type: NotebookNoteLabel) -> Color {
+@inline(__always) func colorFromType(_ type: CourseNoteLabel) -> Color {
     type == .confusing ?
```

Comment:
Feel free to use this code style, more readable and delete the function `colorFromType`

```
enum CourseNoteLabel: String, CaseIterable {
    case confusing = "Confusing"
    case important = "Important"
    case other = "Other"

    var color: Color {
        switch self {
        case .confusing:
            Color(red: 0.682, green: 0.106, blue: 0.122)
        case .important, .other:
            Color(red: 0.055, green: 0.408, blue: 0.702)
        }
    }
}

```